plugin.video.extreme.com/resources/language/Greek/strings.po

```diff
@@ -0,0 +1,69 @@
+# Kodi Media Center language file
+# Addon Name: Extreme Sports
+# Addon id: plugin.video.extreme.com
+# Addon version: 1.0.0
+# Addon Provider: lambda
+
+msgid ""
+msgstr ""
+
+
+msgctxt "#30001"
```

Comment:
Oh, one thing we perhaps also overlooked for the other add-ons:
Please use range 32000-33000 for the ids.
Could you fix that for the already pulled add-ons, too?